features/step_definitions/api-token.js

```diff
@@ -68,11 +58,12 @@ module.exports = function server() {
             method: 'GET',
             auth: {
                 bearer: this.jwt
-            }
+            },
+            json: true
         }).then((response) => {
-            const lastUsed = JSON.parse(response.body)
-                .find(token => token.name === tokenName)
-                .lastUsed;
+            const lastUsed = response.body
+                                     .find(token => token.name === tokenName)
```

Comment:
My instinct was to line the periods up, but it is pretty far. Perhaps one tab would look less weird.